katsdpdisp/data.py

```diff
@@ -122,7 +122,7 @@ def list_files(self, archive_name='primary'):
                     # no create timestamp encoded in filename. Will use stat time
                     created = "*" + time.ctime(int(f[4]))
                 modified = time.ctime(int(f[5]))
-                print str(index).ljust(5),f[1].ljust(65),sz.ljust(9),f[6].ljust(8),created.ljust(25),modified.ljust(25),
+                print(str(index).ljust(5),f[1].ljust(65),sz.ljust(9),f[6].ljust(8),created.ljust(25),modified.ljust(25),)
```

Comment:
If you want the same effect as the trailing comma in Python 2, you need to do this. There are multiple instances of this, but I won't call them all out individually.
```suggestion
                print(str(index).ljust(5),f[1].ljust(65),sz.ljust(9),f[6].ljust(8),created.ljust(25),modified.ljust(25), end=' ')
```

Reply:
There are still more instances of this.

Reply:
I'll change them but it is actually old stale code that wont ever be executed. Just lingers around in case someone like Adriaan want to use old legacy features.